flatflow/nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -458,7 +481,7 @@ def inference_step(self, dataloader_iter, mode):
                 # replace the last appended loss with the outputs dict
                 self.validation_step_outputs[-1] = outputs
         else:
-            if isinstance(self.trainer.test_dataloaders, list) and len(self.trainer.test_dataloaders) > 1:
```

Comment:
I think this line is fine enough, is there a reason for the change?

Reply:
I must have ran ruff format in a wrong way. 
This snippet should be reverted to original code.